### ISSUE TITLE
FileReader requires codec, but Avro spec does not

### ISSUE THREAD

**Issue by gth828r** [ChrisRx/quickavro#25]
According to the 1.8.2 Avro spec at https://avro.apache.org/docs/1.8.2/spec.html:
```
If codec is absent, it is assumed to be "null"
```

When creating a `FileReader` instance with an Avro container file that does not include the codec, this is not what happens.  See https://github.com/ChrisRx/quickavro/blob/master/quickavro/reader.py#L41 for details.

I think if that line is changed to provide a default value of `'null'`, like below, it should resolve the issue:
```py
        self.codec = ensure_str(metadata.get('avro.codec', 'null'))
```

This is mostly important for compatibility with container files created by other Avro implementations, such as the official Apache Java one.

**Comment by gth828r** [comment]
I'm pretty slammed right now, but this was pretty trivial so I created an MR.  I looked into creating a unit test for this, but that was a bit more involved, so I didn't end up creating one.

**Comment by ChrisRx** [comment]
Fixed in #26
